interfaces/dbus/backend.go

```diff
@@ -132,6 +156,68 @@ func addContent(securityTag string, executableSnippets [][]byte, content map[str
 	}
 }
 
+func (b *Backend) setupBusservices(snapInfo *snap.Info, snippets map[string][][]byte) error {
+	snapName := snapInfo.Name()
+
+	content := map[string]*osutil.FileState{}
+	for _, appInfo := range snapInfo.Apps {
+		securityTag := appInfo.SecurityTag()
+
+		// FIXME: layer violation, we really should do that in the
+		//        dbus interface but without PR #2613 its really tricky
```

Comment:
#2613 is now in. Seems okay to fix that in a follow up, though, assuming everything else is golden.